src/promptflow-evals/promptflow/evals/_common/rai_service.py

```diff
@@ -77,6 +73,39 @@ async def ensure_service_availability(rai_svc_url: str, token: str, capability:
         )
 
 
+def generate_payload(normalized_user_text: str, metric: str) -> Dict:
+    """Generate the payload for the annotation request
+
+    :param normalized_user_text: The normalized user text to be entered as the "UserTextList" in the payload.
+    :type normalized_user_text: str
+    :param metric: The evaluation metric to use. This determines the task type, and whether a "MetricList" is needed
+        in the payload.
+    :type metric: str
+    :return: The payload for the annotation request.
+    :rtype: Dict
+    """
+    include_metric = True
```

Comment:
Why do we set `inclue_metric = True` here if it's going to be set False for either option? Is this in expectation of future elif options that would have `include_metric = True`?

Reply:
Yes, the if-statements just below that code set it to false if their conditions are met.